run_scripts/3_generate_outputs.sh

```diff
@@ -2,24 +2,27 @@
 
 set -e
 
-if [ $# -ne 2 ]; then
-    echo "Usage: ./3_generate_outputs.sh <user> <data-root>"
-    echo "Generates the outputs needed downstream from raw data files generated by step 2"
+if [ $# -ne 4 ]; then
+    echo "Usage: ./3_generate_outputs.sh <user> <data-root> <drive-auth> <drive-upload-dir>"
```

Comment:
I think we should work out how we're going to start committing the actual target locations for things like this into the repo somehow as well. Any suggestion?

Reply:
An issue for another PR I think

Reply:
Could you open an issue? I think this one is pretty important, but I agree better in a separate PR